.github/workflows/main.yml

```diff
@@ -84,11 +85,11 @@ jobs:
       - name: Build Docker Image
         if: github.ref != 'refs/heads/main'
         run: |
-          docker build -t "$REGISTRY_HOSTNAME"/"$HOST"/"$IMAGE":${{ env.pr_number }} -f _infra/docker/Dockerfile .
+          docker build -t "$REGISTRY_HOSTNAME"/"$RELEASE_HOST"/"$GAR_REPOSITORY"/"$IMAGE":${{ env.pr_number }} -f _infra/docker/Dockerfile .
```

Comment:
```suggestion
          docker build -t "$REGISTRY_HOSTNAME"/"$GAR_GOOGLE_PROJECT_ID"/images/"$IMAGE":${{ env.pr_number }} -f _infra/docker/Dockerfile .
```

Reply:
The above is an example of the path based on my suggestions and comments above

Reply:
As above

Reply:
Happy to leave `GAR_REPOSITORY` as a var, and that its value is now explicit rather than a GHA secret.